src/applications/facility-locator/tests/e2e/ccpEnabled.cypress.spec.js

```diff
@@ -0,0 +1,67 @@
+import path from 'path';
+import mockFeatureTogglesEnabled from '../fixtures/toggle-ccp-enabled.json';
+import { facilityTypesOptions } from '../../config';
+import { LocationType } from '../../constants';
+
+describe('Facility Search - CCP (community care providers) disabled', () => {
+  before(() => {
+    cy.syncFixtures({
+      constants: path.join(__dirname, '..', '..', 'constants'),
+    });
+  });
+
+  beforeEach(() => {
+    cy.server();
+    cy.route('GET', '/v0/maintenance_windows', []);
+    cy.route('GET', '/v0/facilities/va?*', 'fx:constants/mock-facility-data');
+    cy.route('GET', '/geocoding/**/*', 'fx:constants/mock-geocoding-data');
+  });
+
+  // facilitiesPpmsSuppressCommunityCare flag non existent
+  it('Does render community care option in the dropdown by default', () => {
+    cy.route('GET', '/v0/feature_toggles*', []);
+    cy.visit('/find-locations/');
+
+    cy.injectAxe();
+    cy.axeCheck();
+    cy.get('#facility-type-dropdown')
+      .select(facilityTypesOptions[LocationType.CC_PROVIDER])
+      .should('exist');
+    cy.get('#facility-type-dropdown option').then(options => {
+      const optionsWithoutCCP = [...options].map(o => o.text);
+      expect(
+        optionsWithoutCCP.includes(
+          facilityTypesOptions[LocationType.CC_PROVIDER],
+        ),
+      ).to.eq(true);
+    });
+    cy.get('#facility-type-dropdown option').then(options => {
+      const optionsWithoutCCP = [...options].map(o => o.value);
+      expect(optionsWithoutCCP.includes(LocationType.CC_PROVIDER)).to.eq(true);
+    });
```

Comment:
Same comments apply here:

1. Can we use to.not.include?
2. Delete the 2nd test - not needed IMO